rust/relay/Cargo.toml

```diff
@@ -22,6 +22,7 @@ once_cell = "1.17.1"
 proptest = { version = "1.2.0", optional = true }
 test-strategy = "0.3.0"
 derive_more = { version = "0.99.17", features = ["from"] }
+uuid = { version = "1.3.3", features = ["v4"] }
```

Comment:
Are we using this dependency only for the nonce? I think v4 UUIDs are just random bytes -- maybe we can just use `rand::Rng` or something to generate these?

Reply:
FYI: I'm using UUID for random strings in tests but the portal will accept any random string as a stamp secret.

Reply:
> Are we using this dependency only for the nonce? I think v4 UUIDs are just random bytes -- maybe we can just use `rand::Rng` or something to generate these since it's already a dependency?

Yeah I could just use random hex bytes as well. I'll change it in a follow-up if that is okay.

Reply:
> FYI: I'm using UUID for random strings in tests but the portal will accept any random string as a stamp secret.

These are just for nonces which the portal should never see unless I am missing something.